### DISCUSSION TITLE
not understanding basic math symbols

### DISCUSSION THREAD

**Discussion by tung256** [meatpiHQ/wican-fw#178] [Q&A] [answered]
I see this basic equation for the Hyundai Ioniq 5
![image](https://github.com/user-attachments/assets/ff7febdd-fc0c-4390-9070-e33d7cd1f17a)

i was able to translate it to the proper variable using "B" to use for my Hyundai Ioniq5
for it to become ((B34<8)+B35)/10
what does this "<" mean?  it's not add/subtract/multiply/divide,  what is it?

**Comment by meatpiHQ** [comment, marked as answer]
@tung256 the < here means left shift by 8 bit, wican does not support this yet. Instead you can do this:

`((B34*256)+B35)/10`

Multiple by 256 will left shift by 8

**Reply by tung256** [reply]
yes, i works great now for the Ioniq5!  thanks again
![image](https://github.com/user-attachments/assets/72ab1ae6-f92d-49ee-b40d-45cb18621fed)
{"SOH":**98.6**,"raw":"102E620105FFFB74210F012C01012C1B221B1B1A1B1A1B6C233461620000641A240003DA0424C800253B00000000000026001A1B1A1BAAAA"}
